File: 1. Basic Programming/1. Basics of Input Output/Python/Bricks_game.py
n = int(input())
i = 1
s = 3
while(n>=s):
    i += 1
    s = 3*((i*(i+1))/2)
i -= 1
n_left = n - (3*((i*(i+1))/2))
if(n_left >= i+1):
    print("Motu")
else:
    print("Patlu")

File: 1. Basic Programming/1. Basics of Input Output/C++/Bricks_game.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int n = 0;
    cin>>n;
    int i = 1,s = 3;
    while(n>=s)
    {
        i += 1;
    s = 3*((i*(i+1))/2);
    }
    i -= 1;
   int n_left = n - (3*((i*(i+1))/2));
if(n_left >= i+1)
    cout<<"Motu";
else
    cout<<"Patlu";

}
